blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/CoreConfig.java

```diff
@@ -141,7 +141,8 @@
     STRING_METRICS_DROPPED("false"),
     TENANTIDS_TO_KEEP(""),
 
-    USE_ES_FOR_UNITS("false");
+    USE_ES_FOR_UNITS("false"),
+    ES_UNIT_THREADS("50"); // Should at least be equal to the number of the netty worker threads
 
```

Comment:
@GeorgeJahad hmm, we can do it, but I still think these two configs are independent of each other. The dependence comes into picture because the way we have configured the query handler, the number of threads getting used for querying units in ES, will always be more or less equal to the number of netty worker threads. But what if someone loads another query handler in the future as well, may be UDP query handler, then bumping the ES units query threads makes might make sense, and its no longer dependent on the netty worker threads.